src/api/app/models/staging_workflow.rb

```diff
@@ -7,17 +7,14 @@ def without_staged_requests
   end
   has_many :target_of_bs_requests, through: :project
   has_many :staged_requests, class_name: 'BsRequest', through: :staging_projects
+  has_many :ignored_requests, dependent: :destroy
```

Comment:
You need a `through` here, if not it just returns the model in between and not a list of requests.

Reply:
> > You need a `through` here, if not it just returns the model in between and not a list of requests.
> 
> We agreed that `ignored_requests` should return requests, and not the records of the association table `ignored_requests`.
> 
> I will change the name of the model from `IgnoredRequest` to `RequestIgnoration`. And then `ignored_requests` will return the ignored requests of a staging workflow through `request_ignorations` table.

Done.

Reply:
This discussion is outdated. `ignored_request` is a through association that gives you the request that are ignored